pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -670,7 +670,7 @@ func (c *Cluster) failoverMon(name string) error {
 		return errors.Wrap(err, "failed to place new mon on a node")
 	}
 
-	if c.spec.Network.IsHost() {
+	if  c.spec.Network.IsHost() {
```

Comment:
revert this space, note that vscode would automatically remove these extra spaces for you, so you don't have to think about whitespace much in golang.

Reply:
done (manually 😉 )